src/pybind/mgr/call_home_agent/options.py

```diff
@@ -0,0 +1,5 @@
+# Configurable options for IBM Ceph Call Home Agent
+
+CHES_ENDPOINT = ""
```

Comment:
we sometimes appreviate this module as `CHES` (here), `CHA` (environment vars) or `CCHA` (error messages) - please decide on one of these

Reply:
CHES is Call Home Event streamer, the server with the endpoint where we send reports.
CHA is Call Home Agent , CCHA is Ceph CAll Home Agent, the manager module we are implementing. You are right for this acronym, let's use only CHA